tests/proxy.rs

```diff
@@ -37,7 +38,8 @@ fn test_config() -> config::Config {
         local_xds_path: Some("examples/localhost.yaml".to_string()),
         socks5_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 0),
         inbound_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 0),
-        admin_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 0),
+        admin_addr: SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 0), // Needs to bind to 0.0.0.0 due to the nature of testing environments
```

Comment:
hmm, why is that?

Reply:
I was getting some strange AddressNotAvailable [errors](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_ztunnel/243/test_ztunnel/1600517946004738048) when I bound to localhost instead of 0.0.0.0. I assumed it was due to some docker bridge networking magic in the test environment